deepinv/sampling/samplers.py

```diff
@@ -0,0 +1,347 @@
+import sys
+from collections import deque
+import torch
+from tqdm import tqdm
+from deepinv.physics import Physics
+from deepinv.optim.optim_iterators import *
+from deepinv.optim.prior import Prior
+from deepinv.models import Reconstructor
+from deepinv.optim.data_fidelity import DataFidelity
+from deepinv.sampling.sampling_iterators.sample_iterator import SamplingIterator
+from deepinv.sampling.utils import Welford
+from deepinv.sampling.sampling_iterators import *
+from deepinv.optim.utils import check_conv
+from typing import Union
```

Comment:
To support the modif in the g_statistics line of code
```suggestion
from typing import Union, Callable, Any
```